challenge_storing_groceries/src/challenge_storing_groceries/inspect_shelves.py

```diff
@@ -15,11 +15,12 @@
 class InspectShelves(smach.State):
     """ Inspect all object shelves """
 
-    def __init__(self, robot, cabinet):
+    def __init__(self, robot, cabinet,inspection_result_designator):
```

Comment:
Whitespace after comma (I know it's usually a mess in our files, but I try to be more strict nowadays)